src/SOS/Strike/util.cpp

```diff
@@ -486,6 +537,23 @@ void DisplayDataMember (DacpFieldDescData* pFD, DWORD_PTR dwAddr, BOOL fAlign=TR
                     else
                         ExtOut("%p", SOS_PTR(0));
                     break;
+                case ELEMENT_TYPE_BYREF:
+                {
+                    CLRDATA_ADDRESS methodTable = 0;
+                    DWORD_PTR tgt = ResolveByRefField(pFD, value.ptr, &methodTable);
```

Comment:
If I follow correctly in the case Resolve fails, for example if the pointer in the field points at invalid memory, then we still have DML that tries to pretty print just without the indirection. What if we changed it so that all failure cases from ResolveByRefField returned 0 and the logic here was:
```C++
                    if (tgt)
                    {
                        if (methodTable)
                            DMLOut(DMLByRefValueClass(value.ptr, methodTable, tgt));
                        else
                            DMLOut(DMLByRefObject(value.ptr, tgt));
                    }
                    else
                    {
                        ExtOut("%p", SOS_PTR(value.ptr));
                    }
```

That way the DML link would go away in the failure case rather than trying to evaluate a weird expression.